ui/StatusQ/src/StatusQ/Controls/StatusFeeOption.qml

```diff
@@ -0,0 +1,233 @@
+        case StatusFeeOption.Type.Fast:
+            return qsTr("Fast")
+        case StatusFeeOption.Type.Urgent:
+            return qsTr("Urgent")
+        case StatusFeeOption.Type.Custom:
+            return qsTr("Custom")
+        case StatusFeeOption.Type.Normal:
+        default:
+            return qsTr("Normal")
+        }
+    }
+
+    property string subText
+    property bool showSubText
+
+    property string additionalText
+    property bool showAdditionalText
+
+    property string unselectedText
+
+    property string icon: {
+        switch(type) {
+        case StatusFeeOption.Type.Fast:
+            return Theme.png("wallet/car")
+        case StatusFeeOption.Type.Urgent:
+            return Theme.png("wallet/rocket")
+        case StatusFeeOption.Type.Custom:
+            return Theme.png("wallet/handwrite")
+        case StatusFeeOption.Type.Normal:
+        default:
+            return Theme.png("wallet/clock")
+        }
+    }
+
+    signal clicked()
+
+    font.family: Theme.baseFont.name
+    font.pixelSize: 12
+
+    horizontalPadding: 8
+    verticalPadding: 8
+
+    component AnimatedText: StatusBaseText {
+        id: animatedText
+        verticalAlignment: Qt.AlignVCenter
+        font.family: root.font.family
+        font.pixelSize: root.font.pixelSize
+        wrapMode: Text.WordWrap
+        elide: Text.ElideRight
+
+        onTextChanged: {
+            if (text === "") {
+                return
+            }
+            animate.restart()
+        }
+
+
+        StatusColorAnimation {
+            id: animate
+            target: animatedText
+            fromColor: animatedText.color
+        }
+    }
+
+    property Component subTextComponent: AnimatedText {
+        text: root.subText
+    }
+
+    property Component additionalTextComponent: AnimatedText {
+        text: root.additionalText
+        color: Theme.palette.baseColor1
+    }
+
+    property Component unselectedTextComponent: StatusBaseText {
+        verticalAlignment: Qt.AlignVCenter
+        text: root.unselectedText
+        color: Theme.palette.baseColor1
+        font.family: root.font.family
+        font.pixelSize: root.font.pixelSize
+        wrapMode: Text.WordWrap
+        elide: Text.ElideRight
+    }
+
+    property Component loaderComponent: LoadingComponent {
+        radius: 4
+        height: root.font.pixelSize
+    }
+
+    background: Rectangle {
+        id: background
+        implicitHeight: 84
+        implicitWidth: 101
+        radius: 8
+        border.width: 1
+        border.color: root.selected? Theme.palette.primaryColor1 : Theme.palette.baseColor2
+        color: {
+            if (root.hovered) {
+                return Theme.palette.baseColor2
+            }
+
+            if (root.selected) {
+                return Theme.palette.alphaColor(Theme.palette.baseColor2, 0.1)
+            }
+
+            return Theme.palette.statusAppLayout.backgroundColor
+        }
+
+        MouseArea {
```

Comment:
A better idea is _not_ to include this functionality in `background`; if anyone overrides it, you'll never get the `clicked()` signal

Reply:
This needs addressing; just put it outside of contentItem/background, to the very bottom of this file